scripts/gas-refund-program/persistance/db-persistance.ts

```diff
@@ -47,6 +49,42 @@ export async function fetchVeryLastTimestampProcessed({
   return lastTimestamp;
 }
 
+export async function fetchTotalRefundedPSP(): Promise<BigNumber> {
+  const totalPSPRefunded = await GasRefundParticipation.sum<
+    string,
+    GasRefundParticipation
+  >('refundedAmountPSP');
+
+  return new BigNumber(totalPSPRefunded);
+}
+
+export async function fetchTotalRefundedAmountUSDByAddress(): Promise<{
+  [address: string]: BigNumber;
+}> {
+  const totalRefundedAmountUSDAllAddresses =
+    (await GasRefundParticipation.findAll({
+      attributes: [
+        'address',
+        [
+          Sequelize.fn('SUM', Sequelize.col('refundedAmountUSD')),
+          'totalRefundedAmountUSD',
+        ],
```

Comment:
This `fn` is executed by DB itself, right?

Reply:
yep

Reply:
This is syntactic sugar for 
```sql
SELECT 
   "address",
   sum("refundedAmountUSD") as "totalRefundedAmountUSD"
FROM public."GasRefundParticipations" 
group by "address";
```